### PR TITLE
CORS for sixpack

### PR DESCRIPTION
**Pull request by llonchj**

### CODE REVIEW

**Comment by llonchj** [comment]
this is a different implementation from #256 as it adds CORS for sixpack (not sixpack-web)

File: requirements.txt [hunk 1]
```diff
@@ -5,9 +5,9 @@ Flask-Assets==0.12
 Flask-DebugToolbar==0.7.1
 Flask-SeaSurf==0.1.13
 Flask==0.10
-gevent==1.0.2
+gevent==1.1.2
 gunicorn>=17.5,<=19.4.1
-hiredis==0.1.1
+hiredis==0.1.4
```
Comment (josegonzalez) [inline requirements.txt]: Why these changes?

Comment (llonchj) [inline requirements.txt]: That was only necessary on my side, not for the project. Changes in `requirements.txt` can be removed

**Comment by josegonzalez** [comment]
Any chance you can add cors to the web as well?

**Comment by llonchj** [comment]
@josegonzalez kindly elaborate the requirement for adding CORS to the web

**Comment by josegonzalez** [comment]
There are a few endpoints in the web that are useful - like showing all experiments.

Note: tests fail in this PR, and will need to be fixed before this is merged.

**Comment by llonchj** [comment]
@josegonzalez green tests

**Comment by highercomve** [comment]
Hello, this is gonna be implemented soon?.

I'm trying to use sixpack for an experiment in where I work but we need CORS support, we can't fork the @llonchj branch, but could be better if we have some idea of when this could be in the official repository.

Greetings

**Comment by josegonzalez** [comment]
Waiting on tests to pass. Will merge/release minor once that happens.

**Comment by llonchj** [comment]
:) Thank you!

**Comment by highercomve** [comment]
This is already released? if is released what is the new version number? I don't see this release on the release rst file.

**Comment by kuroisuna** [comment]
Hi, thanks a lot for your hard work! Does this release includes the CORS support to the sixpack web? I would gladly make a PR for that if needed, I've been wanting to request the **/experiments.json** from js for a while now. I can also help with the release in pip.

**Comment by josegonzalez** [comment]
The last release was done before this. I'll make a release in a bit.

**Comment by josegonzalez** [comment]
Done.

**Comment by kuroisuna** [comment]
This works perfectly for the sixpack server, but sixpack-web keeps returning a `no-cors` response, I'm going to make a PR to suppor this (I'm talking about this https://github.com/seatgeek/sixpack/pull/266#issuecomment-278438768) 😉
